docs-chef-io/content/server/upgrades.md

```diff
@@ -48,19 +48,42 @@ The Chef Infra Server 14 upgrade does not automatically reindex existing externa
 
 #### Upgrading to 14.13
 
-Chef Infra Server 14.13 supports external OpenSearch for indexing. To migrate from Elasticsearch to external OpenSearch, you must reindex and reconfigure your database after upgrading to Chef Infra Server 14.13. The duration of this operation will vary depending on your server hardware and the number of node objects on your Chef Infra Server.
+Chef Infra Server 14.13 supports external OpenSearch for indexing. To migrate from Elasticsearch to external OpenSearch, Please follow the migration section below.
 
 #### Steps To Enable External OpenSearch
 
 1. Set the `opensearch['external']` attribute to `true`.
 1. Set the `opensearch['external_url']` attribute to the external OpenSearch URL.
+1. Set the `opscode_erchef['search_queue_mode']` attribute to `batch`.
+1. Set the `opscode_erchef['search_provider']` attribute to `opensearch`.
+1. Set the `opscode_erchef['search_auth_username']` attribute to OpenSearch username.
+1. Set the `opscode_erchef['search_auth_password']` attribute to OpenSearch password.
 
 For example:
 
 ```bash
+opscode_erchef['search_queue_mode'] = 'batch'
+opscode_erchef['search_provider'] = 'opensearch'
 opensearch['external'] = true
 opensearch['external_url'] = "http://127.0.0.1:9200"
+opscode_erchef['search_auth_username'] = 'OPEN_SEARCH_USER'
+opscode_erchef['search_auth_password'] = 'OPEN_SEARCH_PWD'
 ```
+#### Steps To Migrate from Elasticsearch to External OpenSearch
+
+1. Option 1: Copy or move your Elasticsearch OSS data and logs directories to newly installed OpenSearch paths. Follow the steps mentioned at `https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-to-opensearch` for all the steps.
+1. Option 2: Reindex and reconfigure your database after upgrading to Chef Infra Server 14.13. The duration of this operation will vary depending on your server hardware and the number of node objects on your Chef Infra Server.
+
+For example:
+
+```bash
+chef-server-ctl reindex
+chef-server-ctl reconfigure
+```
+
+{{< note >}}
+we support you BYOD, but the upgrades and maintaining of those databases is on the customer.
```

Comment:
context: BYOD is supported from a software perspective. Progress/Chef is not responsible for upgrading and maintaining the external database.